.gitlab/hpsf-gitlab-ci.yml

```diff
@@ -115,3 +115,34 @@ AMD-MI300A:
       - build/result_mi300a.xml
     reports:
       junit: build/result_mi300a.xml
+
+NVIDIA-GRACE-GRACE:
+  extends: .common-setup
+  tags: [uo, aarch64, cpu, grace, neoverse_v2]
+  image: tcclevenger/kokkos-nvidia-cuda:12.6.1-devel-ubuntu24.04
```

Comment:
I don;t quite understand why we are using that custom image rather than start directly from an "official" base image and install whatever packages are missing.

Reply:
We need python in a "pre-build setup" phase, that exists before `script:` section is run (as well as before `before_script:`). I need to familiarize myself with the gitlab yaml interface to see what (if any) sections exist where I can install python before this "pre-build setup" call.

Reply:
Why is that only an issue with this build but not with others?

Reply:
I still don't understand why the NVIDIA image used for GH200 first needed a similar approach but doesn't anymore, see https://github.com/kokkos/kokkos/pull/7714#issuecomment-2621780871.

Reply:
Please investigate further.

Reply:
ping

Reply:
I pushed a different NVIDIA image that has `python3` already.

Reply:
@masterleinad Did you push to `ghcr.io/kokkos/ci-containers/`? or somewhere else?

Reply:
For some reason, this pull request didn't get updated with https://github.com/tcclevenger/kokkos/commit/8e99ddf4f7f826bab610135d66af89f2d0aee1a3.

Reply:
Yeah, it's odd. Closed and reopened and it pulled in the new change.